src/main/java/edu/ucsb/courses/entities/ScheduleItem.java

```diff
@@ -0,0 +1,118 @@
+package edu.ucsb.courses.entities;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+@Entity
+public class  ScheduleItem {
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    @Column(nullable = false)
+    private String courseId;
+    @Column(nullable = false)
+    private String enrollCode;
+    @Column(nullable = false)
+    private String userId;
+    @Column(nullable = false)
+    private Long scheduleId;
+
+
+    //@ManyToOne
+    //@JoinColumn(name = "schedule_id")
+    //private Schedule schedule;
```

Comment:
We can remove unused code instead of commenting it out.

Reply:
Yup